spec/requests/stories_show_spec.rb

```diff
@@ -188,6 +188,7 @@
 
     it "handles invalid slug characters" do
       allow(Article).to receive(:find_by).and_raise(ArgumentError)
+      allow(Article).to receive(:where).and_raise(ArgumentError)
```

Comment:
We used to only do `find_by` but with the `where` clause introduced this keeps the spec working in the same way. Any feedback here to make sure this case is properly covered is appreciated :)